### DISCUSSION TITLE
Didn't Recieved Github Student Developer benifits

### DISCUSSION THREAD

**Discussion by Bharath2954** [community/community#106866] [GitHub Education]
I have applied for student developer pack. It was approved but I didn't get the benifits. May I know how long it will take to get the benifits?

**Comment by Elmore98** [comment]
Same for me. I was rejected first, uploaded another Proof and was accepted. 
I never got any mail, however I can see it online in my account.

My Plan is still free and I have not yet received anything else.

**Comment by aprubio** [comment]
Dear all, 

Same here, I was approved on feb 6 and I still wait to join global campus. What is the estimated waiting time or does anyone has a recommendation of what to do?

Thanks
